### DISCUSSION TITLE
Consiglio: hub, sensori, ZigBee, supporto Home Assistant e Google Home

### DISCUSSION THREAD

**Discussion by officialdavide** [indomus/forum#977] [GENERALE - Discussioni generiche]
Ciao ragazzi/e,
chiedo a voi un consiglio perché, presuppongo, ci siete passati prima di me.

Vorrei chiedervi un parere: vorrei un ecosistema hub e sensori/attuatori (es: hub con sensori porte/finestre), basato su ZigBee.

Mi piacerebbe che abbia queste caratteristiche:
1) ecosistema basato su ZigBee
2) possibilità di pilotare i sensori con l'antenna ConBee 2 + Home Assistant; oltre al pilotaggio via hub
3) ecosistema che si interfacci con Google Home & Google Assistant (non posseggo Alexa, preferisco il sistema di Google)

Ho pensato all'ecosistema di Aqara ma temo non siano integrabili il mondo di Google (ma mi dicono che siano di ottima fattura). Ho letto che molto difficilmente hanno "falsi allarmi" o problemi.

Ho visto anche l'ecosistema di Xiaomi, ma le recensioni non mi hanno entusiasmato (es: falsi allarmi da parte dei sensori di porte/finestre).

Vori ragazzi che cosa mi consigliate?
Inoltre è possibile pilotare un sensore/attuatore sia con il proprio hub e, contemporaneamente, con Home Assistant e ConBee 2?

Un saluto e buona serata!

**Comment by snakuzzo** [comment]
Lascia perdere hub proprietari. Nel tuo caso ti consiglio zigbee2mqtt con la conbee2. Piloti tutto da HA (o dalla gui di zigbee2mqtt... che sarà lui il tuo hub)... e sempre da HA puoi esporre le entità verso i vari assistenti vocali

**Reply by officialdavide** [reply]
Grazie del consiglio!

Ok per l'hub però ti chiedo: un dispositivo potrebbe essere collegato, in contemporanea, da due "antenne zigbee" (es: ConBee2 e hub proprietario)?

Gli Aqara possono essere gestiti con Google Home & Assistant?

Grazie mille per il prezioso consiglio di prima!

**Reply by snakuzzo** [reply]
Un dispositivo può essere collegato ad una sola rete zigbee...non puoi collegare alla Conbee2 e poi anche all'hub proprietario.
Non ricordo su Google Assistant come vengono esposti da HA...devo controllare

**Reply by officialdavide** [reply]
Come "temevo" un dispositivo può collegarsi ad una sola rete alla volta ma grazie mille, Diego, per il riscontro!
